### PR TITLE
Move HNC to its own repo

### PR DESCRIPTION
**Pull request by adrianludwin**
Update all links, and mark the Tenant Operator as being deprecated now
that it's not used by VC or being developed anymore.

### CODE REVIEW

**Comment by adrianludwin** [comment]
See modified versions here: https://github.com/adrianludwin/multi-tenancy/tree/last-update

* [MTWG README](https://github.com/adrianludwin/multi-tenancy/tree/last-update/README.md)
* [HNC README](https://github.com/adrianludwin/multi-tenancy/tree/last-update/incubator/hnc)
* [HNC user guide](https://github.com/adrianludwin/multi-tenancy/tree/last-update/incubator/hnc/docs/user-guide)

/assign @rjbez17 
/assign @yiqigao217

**Comment by adrianludwin** [comment]
Also cc @tashimi and @Fei-Guo for the changes to the front page.

/assign @tashimi 
/assign @Fei-Guo

File: incubator/hnc/README.md [hunk 1]
```diff
@@ -53,9 +58,8 @@ objects you can't afford to lose (e.g. that aren't stored in a Git repository).
 All HNC issues are assigned to an HNC milestone. So far, the following
 milestones are defined or planned:
 
-* v1.0 - Likely late Q2 or early Q3 2021: HNC recommended for production use
-* [v0.9](https://github.com/kubernetes-sigs/multi-tenancy/milestone/21): move
-  HNC to its own repo; continued stability improvements.
+* ***Post v0.8: please visit the [new
```
Comment (yiqigao217) [inline incubator/hnc/README.md:61]: (Comment on line 35-36 `Getting started and learning more` section - "The latest version of HNC is v0.8.0")
We may mention this is the latest version in this repo and see the latest HNC in the new repo there too.

**Review by yiqigao217 — COMMENTED** [review]
One nit otherwise lgtm

File: incubator/hnc/README.md (hunk 1 above)
Comment (adrianludwin) [reply incubator/hnc/README.md:64]: I _think_ I've fixed this - lgty?

**Review by yiqigao217 — COMMENTED** [review]
/lgtm
/hold
for other reviewers.

**Comment by rjbez17** [comment]
lgtm to me as well. I'll let Tasha or Fei remove the hold.

**Pull request merged**